Sources/MongoSwiftSync/Exports.swift

```diff
@@ -97,7 +97,6 @@
 @_exported import struct MongoSwift.ServerOpeningEvent
 @_exported import struct MongoSwift.ServerSelectionError
 @_exported import struct MongoSwift.Symbol
-@_exported import struct MongoSwift.TLSOptions
```

Comment:
for the sake of the changelog, maybe we should file a ticket that tracks this change and just close it when this gets merged in.

Reply:
good point. I opened [SWIFT-753](https://jira.mongodb.org/browse/SWIFT-753)